packages/@coorpacademy-components/src/template/skill-detail/test/fixtures/default.js

```diff
@@ -0,0 +1,74 @@
+import cardsGrid from '../../../../organism/cards-grid/test/fixtures/catalog';
+
+const cards = cardsGrid.props;
+const allCourses = cards;
+const onGoingCourses = cards.list;
+
+export const defaultProps = {
+  title: 'Conflict management',
+  description:
+    'Conflict management is the skill of handling disagreements constructively through effective communication, active listening, and finding mutually beneficial resolutions. It helps navigate tensions, foster collaboration, and maintain a positive environment by preventing conflicts from escalating.',
+  metrics: {
+    score: 60,
+    content: 89,
+    questionsToReview: 25,
+    contentCompleted: 50
+  },
+  focused: true,
+  availableForReview: true,
+  ongoingCourses: onGoingCourses,
+  skillIncludedCourses: allCourses,
+  filters: {
+    options: [
+      {
+        name: 'All content',
+        value: 'ALL',
+        selected: true
+      },
+      {
+        name: "5' learning",
+        value: 'microlearning',
+        selected: false
+      },
+      {
+        name: 'Courses',
+        value: 'learner',
+        selected: false
+      },
+      {
+        name: 'Video',
+        value: 'scorm',
+        selected: false
+      }
+    ],
+    onChange: value => console.log(value)
+  },
+  sorting: {
```

Comment:
(that will simplify to have similar behavior than skill picker, where UI is updated when we play with the button/selects without having to rerender the component)

Reply:
As discussed yesterday, current sorting and filter structure is based on Catalog page, and it makes managing and interacting with the $stateParams easier instead of hardcoding all options in components